ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlan-interfaces.j2

```diff
@@ -1,156 +1,166 @@
+{%     if vlan_interfaces[vlan_interface].arp_aging_timeout is arista.avd.defined %}
    arp aging timeout {{ vlan_interfaces[vlan_interface].arp_aging_timeout }}
-{%         endif %}
-{%             if vlan_interfaces[vlan_interface].ip_proxy_arp is defined and vlan_interfaces[vlan_interface].ip_proxy_arp == true %}
+{%     endif %}
+{%     if vlan_interfaces[vlan_interface].ip_proxy_arp is arista.avd.defined(true) %}
    ip proxy-arp
-{%             endif %}
-{%         if vlan_interfaces[vlan_interface].ip_address is defined %}
+{%     endif %}
+{%     if vlan_interfaces[vlan_interface].ip_address is arista.avd.defined %}
    ip address {{ vlan_interfaces[vlan_interface].ip_address }}
-{%             if vlan_interfaces[vlan_interface].ip_address_secondaries is defined and vlan_interfaces[vlan_interface].ip_address_secondaries is not none %}
-{%                 for ip_address_secondary in vlan_interfaces[vlan_interface].ip_address_secondaries %}
+{%         if vlan_interfaces[vlan_interface].ip_address_secondaries is arista.avd.defined %}
+{%             for ip_address_secondary in vlan_interfaces[vlan_interface].ip_address_secondaries %}
    ip address {{ ip_address_secondary }} secondary
-{%                 endfor %}
-{%             endif %}
+{%             endfor %}
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].ip_virtual_router_address is defined %}
+{%     endif %}
+{%     if vlan_interfaces[vlan_interface].ip_virtual_router_address is arista.avd.defined %}
    ip virtual-router address {{ vlan_interfaces[vlan_interface].ip_virtual_router_address }}
-{%         endif %}
-{%         if vlan_interfaces[vlan_interface].ip_address_virtual is defined %}
+{%     endif %}
+{%     if vlan_interfaces[vlan_interface].ip_address_virtual is arista.avd.defined %}
    ip address virtual {{ vlan_interfaces[vlan_interface].ip_address_virtual }}
-{%         endif %}
-{%         if vlan_interfaces[vlan_interface].ip_helpers is defined and vlan_interfaces[vlan_interface].ip_helpers is not none %}
-{%            for ip_helper in vlan_interfaces[vlan_interface].ip_helpers | arista.avd.natural_sort %}
-   ip helper-address {{ ip_helper }} {% if vlan_interfaces[vlan_interface].ip_helpers[ip_helper].vrf is defined and vlan_interfaces[vlan_interface].ip_helpers[ip_helper].vrf is not none %}vrf {{vlan_interfaces[vlan_interface].ip_helpers[ip_helper].vrf }} {% endif %} {% if vlan_interfaces[vlan_interface].ip_helpers[ip_helper].source_interface is defined and vlan_interfaces[vlan_interface].ip_helpers[ip_helper].source_interface is not none %}source-interface {{vlan_interfaces[vlan_interface].ip_helpers[ip_helper].source_interface }}{% endif %}
-
-{%            endfor %}
-{%         endif %}
-{%         if vlan_interfaces[vlan_interface].ipv6_enable is defined and vlan_interfaces[vlan_interface].ipv6_enable == true %}
+{%     endif %}
+{%    for ip_helper in vlan_interfaces[vlan_interface].ip_helpers | arista.avd.natural_sort %}
```

Comment:
Missing one space for indentation between line 42 and line 49

Reply:
Fixed with  07d5adb6